### PR TITLE
feat(config): add Inteset Door/Window Sensor

### PR DESCRIPTION
**Pull request by cdnninja**
Added Inteset Door/Window Sensor.  I can't find a real device name for the life of me.

### CODE REVIEW

**Comment by cdnninja** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x039a/doorWindow.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+	"manufacturer": "Inteset",
+	"manufacturerId": "0x039a",
+	"label": "Inteset Door/Window Sensor",
+	"description": "Door/Window Sensor",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x039a/doorWindow.json]: @blhoward2 what should we do about these?

File: packages/config/config/devices/0x039a/doorWindow.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Basic Set off Delay Time",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x039a/doorWindow.json]:
```suggestion
			"label": "Basic Set: Off Delay Time",
```

File: packages/config/config/devices/0x039a/doorWindow.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 0
+		},
+		{
+			"#": "2",
+			"label": "Basic Set Level",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x039a/doorWindow.json]:
```suggestion
			"label": "Basic Set: Level",
```

File: packages/config/config/devices/0x039a/doorWindow.json [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100
+		},
+		{
+			"#": "3",
+			"label": "Led Indicator",
+			"description": "Indicator on open close event",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x039a/doorWindow.json]:
```suggestion
			"label": "LED Indicator: Open/Close",
```

File: packages/config/config/devices/0x039a/doorWindow.json [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+			]
+		},
+		{
+			"#": "5",
+			"label": "Battery Report Interval",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 1080,
+			"defaultValue": 480
```
Comment (AlCalzone) [inline packages/config/config/devices/0x039a/doorWindow.json]: Are these really seconds? Battery reports every 8 minutes sounds like a sure way to drain the battery quickly.

Comment (AlCalzone) [reply packages/config/config/devices/0x039a/doorWindow.json]: Edit: This seems to be minutes, not seconds. See the manual you linked.

File: packages/config/config/devices/0x039a/doorWindow.json [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+			"defaultValue": 0
+		},
+		{
+			"#": "2",
+			"label": "Basic Set Level",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100
```
Comment (AlCalzone) [inline packages/config/config/devices/0x039a/doorWindow.json]: 1. This isn't %. Basic Set commands go from 0 to 99.
2. The linked manual states that 100 maps to an "ON" command (0xff). This should be represented as an option

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x039a/doorWindow.json (hunk 6 above)
Comment (cdnninja) [reply packages/config/config/devices/0x039a/doorWindow.json]: Done.

Comment (cdnninja) [reply packages/config/config/devices/0x039a/doorWindow.json]: Looks like the checker isn't happy having options in addition to manual entry.   I don't think this is a partial parameter though rather the value it passes to the second association group?

Comment (blhoward2) [reply packages/config/config/devices/0x039a/doorWindow.json]: That's not the issue. You can have an option with manual entry. You just only include allowManualEntry if it's false. It defaults to true. 

```
Parameter #2: allowManualEntry must be false or omitted! (ZW0340)
10
```

File: packages/config/config/devices/0x039a/doorWindow.json (hunk 1 above)
Comment (blhoward2) [reply packages/config/config/devices/0x039a/doorWindow.json]: I've had a few like this and I searched for the actual model number or part number. It's there is truly no model number, we'd just have to repeat it.

File: packages/config/config/devices/0x039a/doorWindow.json (hunk 6 above)
Comment (cdnninja) [reply packages/config/config/devices/0x039a/doorWindow.json]: Gotcha!  I will correct.

File: packages/config/config/devices/0x039a/doorWindow.json (hunk 1 above)
Comment (cdnninja) [reply packages/config/config/devices/0x039a/doorWindow.json]: Found the model number from what I can tell.  I have since included it.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**